docs/xcm/building-with-xcm/erc20-cross-chain-compatible.md

```diff
@@ -0,0 +1,97 @@
+Let's call this new wrapped token xcBURRITO.
+This xcBURRITO takes the address of the underlaying token (BURITTO) and the newly created asset's address as a constructor parameters. And we’ll set values for all the other required parameters, notice that we have to include ERC20Permit constructor call because xcBURRITO is now a parent for BURITTO.
+```Solidity
+constructor(IERC20 burrito)
+   ERC20("Wrapped Burrito", "xcBUR")
+   ERC20Permit("Wrapped Burrito")
+   ERC20Wrapper(burrito)
+{}
+```
+Since we can't use ERC20Wrapper out of box we'll override it to use **XC20+** interface
+```Solidity
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Wrapper.sol";
+import "./Burrito.sol";
+
+contract XcBurrito is Xc20Plus, ERC20Wrapper, BURITTO{
+    constructor(IERC20 burrito)
+    Xc20Plus("Wrapped Burrito", "xcBUR")
+    ERC20Permit("Wrapped Burrito")
+    ERC20Wrapper(burrito)
+    {}
+
+    function _mint(address _to, uint256 _amount)
+        internal
+        override(ERC20)
+    {
+        // add here your pre-mint hooks if needed
+
+        require(
+            IERC20Plus(xcBurrito).mint(_to, _amount), "Minting xc token failed"
+        );
+
+        // add here your post-mint hooks if needed
+    }
+}
+```
+Check the full example on the Github repo under [examples](https://github.com/AstarNetwork/astar-frame).
+
+## Procedures
+### 1. Create XC20 asset
+Follow the documentation on how to [Create XC20 Assets](https://docs.astar.network/docs/xcm/building-with-xcm/create-xc20-assets).
+
+### 2. Deploy XcBurrito.sol smart contract
+To deploy XcBurrito contract you need 2 input parameters
+- Burrito ERC20 token address (H160)
+- **XC20** asset address (H160)
+    - Follow instructions on how to [Generate XC20 Address](https://docs.astar.network/docs/xcm/integration/tools). If your asset_id is 17 (=0x11), your EVM address will be `0xffffffff00000000000000000000000000000011`
+
+### 3. Transfer xc asset ownership to the XcBurrito smart contract
+To enable **xcBurrito** contract to be able to mint/burn you need to call `setTeam()` extrinsic on pallet-assets. Configure the `issuer` and `admin` to be **xcBurrito**. This will enable the contract to issue wrapped **XC20** assets. Since the EVM contract has 24 bytes (H160), we need to transform EVM address into SS58 format to be able to call this extrinsic.
+![polkadot-js-ss-1](https://user-images.githubusercontent.com/34627453/182795050-f10dad6e-dbd9-4ac9-b526-d51a66503cb9.png)
+
+You can renounce ownership of the **XC20** by calling `transferOwnership()` extrinsic on pallet-assets. Note that after this, you won't be able to
+change `issuer` or `admin` anymore so make sure those are set correctly.
+
+![polkadot-js-ss-2](https://user-images.githubusercontent.com/34627453/182795097-d145073b-cdf0-4c28-916f-66abb5c68abc.png)
```

Comment:
Other images in the docs are local in the [local img folder](https://github.com/AstarNetwork/astar-docs/tree/doc/erc20-xc-compatible/docs/xcm/building-with-xcm/img) so these two pics could be put likewise. Currently images are numbered 1, 2, 3, 4 but personally I find this very obscure, maybe rather have them named erc20-cross-chain-compatible-1.png and erc20-cross-chain-compatible-2.png 

I can do this to be honest I do not know hot to add files to your PR? Perhaps after you merge PR with another PR?

Reply:
You just need to push to the same branch I'm working on, that's it.